### PR TITLE
First trying to read file content for Searching the samples from a file

### PR DESCRIPTION
**Pull request by hessikaveh**
Making a pull request as it is easier to follow comments and debugging here:
For the moment when I submit the file the page gets updated and the content of the file is shown but the rest of the tables on the page become empty!
@justinasr @pgunnell

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -336,6 +336,31 @@ def run_flask():
...
+            content = 'No File Selected!'
+            return redirect(request.url)
+        file = request.files['file']
+        if file.filename == '':
+            content = 'Wrong Format File(use .txt)!'
+            return redirect(request.url)
+        if file and allowed_file(file.filename):
+            filename = secure_filename(file.filename)
+            content = file.read()
+        return render_template("index.html", text = content)
```
Comment (pgunnell) [inline main.py]: I guess you see only the content of the file because of this line, no?

**Comment by pgunnell** [comment]
Wow, thanks Sam! great..I left you one comment on the code, related to your question.

File: main.py (hunk 1 above)
Comment (justinasr) [reply main.py]: Paolo is right. If you want to see other things too, you have to pass other variables into `render_template` function. At the moment you are trying to render index.html template which is the initial page of Samples. If you look at `index()` function, it's return statement in particular, you'll see that a bunch of other things should be passed too: campaigns, campaign_groups and pwgs. These are lists that are rendered in the page as tables. Campaigns are fetched from the sqlite database.
Note: after latest changes, I think only campaign_groups are necessary as initial page does not show separate campaigns anymore. This needs to be updated.

Comment (hessikaveh) [reply main.py]: Thanks a lot, it worked by changing the render_template call to include the other arguments from the index() function. I will try to find how to implement a search query for each line of the input text file to the database as the next step.

**Pull request closed without merging**